src/sentry/static/sentry/app/components/charts/components/tooltip.jsx

```diff
@@ -31,8 +35,7 @@ function getFormatter({filter, isGroupedByDate, truncate, utc}) {
         .filter(getFilter)
         .map(
           s =>
-            `<div>${s.marker} ${truncationFormatter(s.seriesName, truncate)}:  ${s
-              .data[1]}</div>`
+            `<div>${s.marker} ${truncationFormatter(s.seriesName, truncate)}:  ${formatDelimeters(s.data[1])}</div>`
```

Comment:
Can we call it something like `valueFormatter` that accepts a 2nd parameter that enables/disables formatting (similar to how `truncate` is used except this will be a boolean).